src/utils/validators.js

```diff
@@ -595,69 +604,64 @@ const validateLocationType = (locationType, value) => {
     case "title": {
       // Title is too short
       if (value.length < LOCATION_TITLE_MIN_LENGTH) {
-        errorMessage = `Title cannot be empty.`
+        errorMessage = `Label cannot be empty.`
       }
       // Title is too long
       if (value.length > LOCATION_TITLE_MAX_LENGTH) {
-        errorMessage = `Title should be shorter than ${LOCATION_TITLE_MAX_LENGTH} characters.`
+        errorMessage = `Label should be shorter than ${LOCATION_TITLE_MAX_LENGTH} characters.`
       }
       break
     }
     case "maps_link": {
+      if (value.length > 0 && !value.startsWith("https://")) {
```

Comment:
For `www` we should not, because it will not correctly format the link for the end-user, so we definitely need to specify the protocol. For `http`, I decided that generally since our sites are in HTTPS, we should also make links go to other HTTPS sites as much as possible. At least for Google Maps and OneMap, they are in HTTPS so this should not affect them.

Reply:
to double check - this is **new** functionality that wasn't previously present right? i actually don't mind this (and i agree w/ your reasoning here), just wanted to confirm this